Source/WebCore/css/PropertySetCSSStyleDeclaration.cpp

```diff
@@ -153,7 +153,7 @@ unsigned PropertySetCSSStyleDeclaration::length() const
 {
     unsigned exposed = 0;
     for (unsigned i = 0; i < m_propertySet->propertyCount(); i++) {
-        if (isCSSPropertyExposed(m_propertySet->propertyAt(i).id()))
+        if (isExposed(m_propertySet->propertyAt(i).id()))
```

Comment:
`StyleProperties` would be another good one to add an iterator for at some point.

Reply:
Great idea.